src/config/ssd.md

```diff
@@ -0,0 +1,153 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If DISC-GRAN (discard granularity) and DISC-MAX (discard max bytes) columns are
+non-zero, that means the block device has TRIM support. If your solid state
+drive partition does not show TRIM support, please verify that you chose a file
+system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. For example, if your
+/ directory is on a SSD, run the following command to run TRIM one-shot:
+
+> Note: To run fstrim on F2FS file systems, you need to be running kernel 4.19
+> or greater.
+
+```
+# fstrim /
+```
+
+We can automate running TRIM by using cron or by adding the `discard` option to
+fstab.
+
+## Periodic TRIM with Cron
+
+Add the following lines to `/etc/cron.daily/fstrim`.
+
+```
+#!/bin/sh
+
+fstrim /
+```
+
+Finally, set the execution bit for the script:
+
+```
+# chmod u+x /etc/cron.daily/fstrim
+```
+
+## Continuous TRIM with Fstab discard
+
+> Note: You can use either continuous or periodic TRIM, but usage of continuous
+> TRIM is discouraged if you have an SSD that doesn't handle NCQ correctly.
+> Refer to the
+> [blacklist](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/ata/libata-core.c#n4522).
+
+Edit `/etc/fstab` and add the `discard` option to block devices that need TRIM.
+
+For example, if `/dev/sda1` was an SSD partition, formatted as ext4, and mounted
+to `/` with discard options, add the `discard` option:
```

Comment:
s/to/at/
Remove all text from "with discard" up to, but not including, the colon.